src/sentry/static/sentry/app/views/settings/project/projectReleaseTracking.jsx

```diff
@@ -138,13 +125,18 @@ class ProjectReleaseTracking extends AsyncView {
           <PanelHeader>{t('Client Configuration')}</PanelHeader>
           <PanelBody disablePadding={false} flex>
             <p>
-              {tct('Start by binding the [release] attribute in your application', {
-                release: <code>release</code>,
-              })}
+              {tct(
+                'Start by binding the [release] attribute in your application, take a look at [link] to see how to configure this for the SDK you are using.',
+                {
+                  link: (
+                    <a href="https://docs.sentry.io/workflow/releases/?platform=javascript#tag-errors">
```

Comment:
wonder if we should try to use `project.platform` here?

Reply:
Not sure if we should do this, we "only" have Python, JS, C# and Rust in the docs here yet.
I suppose there are more platforms in Sentry.